questions.js

```diff
@@ -1,232 +1,17 @@
-
-const nonAlphaNum = '[^(a-z|A-Z|0-9)]*'
-const tossupMarkers = new RegExp('\\s*TU' + nonAlphaNum + '[0-9]+' + nonAlphaNum)
+// const nonAlphaNum = '[^(a-z|A-Z|0-9)]*'
+// const markers = ['\\s*TU' + nonAlphaNum + '[0-9]+' + nonAlphaNum,
+// '[0-9]+' + nonAlphaNum + 'TU' + nonAlphaNum]
+// const tossupMarkers = new RegExp(markers.join('|'))
+// const tossupMarkers = new RegExp('\\s*TU' + nonAlphaNum + '[0-9]+' + nonAlphaNum)
+const tossupMarkers = new RegExp("^[^(a-z|A-Z)]*(?:TU|Tossup)[^(a-z|A-Z)]+", 'm') //?: makes the group non-capturing so it's not included in split
 const bonusMarkers = /\s*B(1|2)\s*/ //"B#"
 //^ add to them using | 
-const questionPattern = /(.|\s)*?(?=\s+[^a-z]+$)/ //(.|\s) = anything or a whitespace, n? = contains 0 or one occurence of n (non-greedy matching)
-const answerPattern = /(?<=\s+)[^a-z]+$/
+const questionPattern = /(.|\s)*?(?=\s+[^a-z_]+$)/ //(.|\s) = anything or a whitespace, n? = contains 0 or one occurence of n (non-greedy matching)
```

Comment:
In the future: capture the answer first, then remove that from the text to get the question